make error

When I clone the code locally, I get an error using the make method.
```
go: finding module for package dtweave.io/goframe/samples/myoptions
go: finding module for package dtweave.io/goframe/app
go: finding module for package dtweave.io/goframe/app/version
go: finding module for package dtweave.io/goframe/app/flag
github.com/dtweaveio/goframe/app imports
        dtweave.io/goframe/app/flag: cannot find module providing package dtweave.io/goframe/app/flag: unrecognized import path "dtweave.io/goframe/app/flag": https fetch: Get "https://dtweave.io/goframe/app/flag?go-get=1": dial tcp: lookup dtweave.io: no such host
github.com/dtweaveio/goframe/app imports
        dtweave.io/goframe/app/version: cannot find module providing package dtweave.io/goframe/app/version: unrecognized import path "dtweave.io/goframe/app/version": https fetch: Get "https://dtweave.io/goframe/app/version?go-get=1": dial tcp: lookup dtweave.io: no such host
github.com/dtweaveio/goframe/samples imports
        dtweave.io/goframe/app: cannot find module providing package dtweave.io/goframe/app: unrecognized import path "dtweave.io/goframe/app": https fetch: Get "https://dtweave.io/goframe/app?go-get=1": dial tcp: lookup dtweave.io: no such host
github.com/dtweaveio/goframe/samples imports
        dtweave.io/goframe/samples/myoptions: cannot find module providing package dtweave.io/goframe/samples/myoptions: unrecognized import path "dtweave.io/goframe/samples/myoptions": https fetch: Get "https://dtweave.io/goframe/samples/myoptions?go-get=1": dial tcp: lookup dtweave.io: no such host
make: *** [tidy] Error 1

```

--- a/app/app_build.go
+++ b/app/app_build.go
@@ -6,8 +6,8 @@ import (
 	"os"
 	"strings"
 
-	"dtweave.io/goframe/app/flag"
-	"dtweave.io/goframe/app/version"
+	"github.com/dtweaveio/goframe/app/flag"
+	"github.com/dtweaveio/goframe/app/version"
 	"github.com/fatih/color"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"

--- a/app/options.go
+++ b/app/options.go
@@ -1,6 +1,6 @@
 package app
 
-import "dtweave.io/goframe/app/flag"
+import "github.com/dtweaveio/goframe/app/flag"
 
 type Options interface {
 	// Flags returns flags for sub gateway

--- a/go.mod
+++ b/go.mod
@@ -5,8 +5,6 @@ go 1.19
 require (
 	github.com/fatih/color v1.15.0
 	github.com/gosuri/uitable v0.0.4
-	github.com/onsi/ginkgo/v2 v2.12.0
-	github.com/onsi/gomega v1.27.10
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.16.0
@@ -14,10 +12,6 @@ require (
 
 require (
 	github.com/fsnotify/fsnotify v1.6.0 // indirect
-	github.com/go-logr/logr v1.2.4 // indirect
-	github.com/go-task/slim-sprig v0.0.0-20230315185526-52ccab3ef572 // indirect
-	github.com/google/go-cmp v0.5.9 // indirect
-	github.com/google/pprof v0.0.0-20210407192527-94a9f03dee38 // indirect
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/magiconair/properties v1.8.7 // indirect
@@ -31,10 +25,8 @@ require (
 	github.com/spf13/cast v1.5.1 // indirect
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/subosito/gotenv v1.4.2 // indirect
-	golang.org/x/net v0.14.0 // indirect
 	golang.org/x/sys v0.11.0 // indirect
 	golang.org/x/text v0.12.0 // indirect
-	golang.org/x/tools v0.12.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )

--- a/samples/myapp.go
+++ b/samples/myapp.go
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"dtweave.io/goframe/app"
-	"dtweave.io/goframe/samples/myoptions"
+	"github.com/dtweaveio/goframe/app"
+	"github.com/dtweaveio/goframe/samples/myoptions"
 )
 
 func main() {

--- a/samples/myoptions/myoptions.go
+++ b/samples/myoptions/myoptions.go
@@ -1,6 +1,6 @@
 package myoptions
 
-import myflag "dtweave.io/goframe/app/flag"
+import myflag "github.com/dtweaveio/goframe/app/flag"
 
 type MyOptions struct {
 	MysqlOptions *MysqlOptions `json:"mysql" mapstructure:"mysql"`

